### ISSUE TITLE
Gitako Not showing  after a branch is force pushed

### ISSUE THREAD

**Issue by ShinChven** [EnixCoda/Gitako#242]
Gitako Not showing  after a branch is force pushed.

https://github.com/ShinChven/react-virtuoso-example

**Comment by EnixCoda** [comment]
@ShinChven 

Please make sure you've upgraded to the latest version `v3.7.3`. It contains an important adoption fix to the latest GitHub features.

Gitako loads properly in the repo and continues to work as I navigate to files and tabs, during my test.

By `force push`, does it mean Gitako would break when you run git force push while the tab is already open?

**Comment by ShinChven** [comment]
@EnixCoda 
Thank you for you reply! 

Chrome has upgraded the extension to v3.7.4, the problem does not exist anymore. 

In my case, the extension was working with this repo while I first pushed the master branch. I don't remember if I closed the tab later, It just didn't work after I force-pushed the branch. The extension didn't show floating icon nor sidebar. I tried reload the web or restart chrome, it didn't work.
